apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-expiration/vo-settings-expiration.component.ts

```diff
@@ -20,6 +20,7 @@ export class VoSettingsExpirationComponent implements OnInit {
   @HostBinding('class.router-component') true;
 
   expirationAttribute: Attribute;
+  loading = false;
```

Comment:
```suggestion
  expirationAttribute: Attribute = { id: -1, beanName: 'Attribute', value: null };
  loading = false;
```
For the correct behavior, we need to add in `expiratoin-settings.component.ts` this update too:
```
this.currentConfiguration = this.unParseAttrValue(
   this.expirationAttribute.value as ExpirationAttrValue,
);
```
And maybe we can remove this and also initialConfiguration setting in ngOnInit here.